var/spack/repos/builtin/packages/tcl/package.py

```diff
@@ -70,6 +70,15 @@ def symlink_tclsh(self):
     # ========================================================================
     # Set up environment to make install easy for tcl extensions.
     # ========================================================================
+    @property
+    def command(self):
+        """Returns the tclsh command.
+
+        :returns: The tclsh command
+        :rtype: Executable
+        """
+        path = os.path.realpath(os.path.join(self.prefix.bin, 'tclsh'))
```

Comment:
You should be able to use `self.prefix.bin.tclsh` instead of `join_path` here if you want to make it simpler.

Reply:
TIL of idiomatic Spack :)